users/templates/users_detail.html

```diff
@@ -32,8 +32,8 @@ <h1>
 
   </div>
     <form class="form-horizonal" action="{% url 'custom-user-update-by-admin' pk=user_detail.pk %}" method="POST">
-      {% csrf_token %}
-      {% render_form serializer template_pack='rest_framework/vertical' %}
+        {% csrf_token %}
```

Comment:
2-spaces "tab" is ok in HTML, please align.

Reply:
:+1: